sql/messaging_system_derived/cfr_users_daily_v1/query.sql

```diff
@@ -0,0 +1,46 @@
+WITH windowed AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    impression_id,
+    client_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    --
+    -- For all dimensions, we use the mode of observed values in the day.
+    udf.mode_last(ARRAY_AGG(release_channel) OVER w1) AS release_channel,
+    udf.mode_last(ARRAY_AGG(addon_version) OVER w1) AS addon_version,
+    udf.mode_last(ARRAY_AGG(metadata.geo.country) OVER w1) AS country,
+    udf.mode_last(ARRAY_AGG(version) OVER w1) AS version,
```

Comment:
I picked out the dimensions that seem to be more-or-less stable per client to include here.